CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 # Unreleased
 
+## 1.17.1
+
+- Remove exception hack (#1307)
```

Comment:
isn't this internal, meaning doesn't need to be in the release notes?

Reply:
Fair enough.